src/mmw/apps/modeling/models.py

```diff
@@ -79,7 +79,7 @@ class Meta:
     name = models.CharField(
         max_length=255)
     project = models.ForeignKey(Project, related_name='scenarios')
-    current_condition = models.BooleanField(
+    is_current_conditions = models.BooleanField(
```

Comment:
Conceptually, I don't see "current conditions" as different from other model simulations. Maybe this will become more clear as we add alternative model simulations. Thanks for explaining.